### PR TITLE
DISPATCH-1913: Fix command line tools

### PR DESCRIPTION
**Pull request by astitcher**
This avoids deprecated behaviour surfaced by a recent change to Proton

### CODE REVIEW

File: tools/qdstat.in [hunk 1]
```diff
@@ -44,18 +44,17 @@ from qpid_dispatch_internal.compat import UNICODE
 def get(obj, attr):
     return getattr(obj, attr, None)
 
-class BusManager(Node):
+class BusManager():
```
Comment (kgiusti) [inline tools/qdstat.in:47]: make this a subclass of object:  "class BusManager(object):"

Comment (astitcher) [reply tools/qdstat.in:47]: Is that necessary for python 3? Also might want that change consistently through the file if you think it's necessary.

Comment (astitcher) [reply tools/qdstat.in:47]: I guess I meant shouldn't this just be: ```class BusManager:```  - I'm pretty sure this works even in python 2.7. What you say was true for much earlier versions of python.

Comment (jiridanek) [reply tools/qdstat.in:47]: I'm with Ken here. https://docs.python.org/2.7/glossary.html#term-new-style-class and https://docs.python.org/2.7/reference/datamodel.html#newstyle state that you have to inherit from _object_ to get a new-style class. That is true even in Python 2.7.

Comment (astitcher) [reply tools/qdstat.in:47]: @jiridanek Thanks for the pointers! I was wrong about new style classes. I did a quick grep and there are a bunch of old style classes in this code base. For leaf classes (actually such as this one) I'm not sure it's that important as the differences seem to mostly be about metaclass and inheritance features.
I'll change this PR before I merge it though.

**Pull request closed without merging**